scheduler/src/cook/components.clj

```diff
@@ -338,6 +338,9 @@
   (try
     ; Note: If the mount/start-with-args fails to initialize a defstate S, and/or you get weird errors on startup,
     ; you need to require S's namespace with ns :require. 'ns :require' is how mount finds defstates to initialize.
+    ;
+    ; If you get an error about "Can't embed object in code, maybe print-dup not defined: clojure.lang.Delay"
+    ; The issue is about metatransaction.core seems to be incompatible with mount. It cannot be in the dependency tree.
```

Comment:
This comment seems overly specific in blaming `metatransaction.core` for the error. What if we have another namespace that causes the same error when `require`'d into the dependency tree? How did you trace the blame back to `metatransaction.core`? The first sentence on 343 has a typo, so it needs to be reworded anyway.

Maybe just mention the error and link to issue #1370 instead. I probably should have done something like that when I opened the issue.

Reply:
I was able to create the error only adding a dependency to metatransaction.core, and was able to make it go away by deleting the reference. The issue you found adding progress was because of this dependency chain: cook.progress -> cook.tools -> metatransaction.core. I reworded it.